Library/Homebrew/patch.rb

```diff
@@ -64,9 +64,9 @@ def contents; end
 
   def apply
     data = contents.gsub("HOMEBREW_PREFIX", HOMEBREW_PREFIX)
-    cmd = "/usr/bin/patch"
+    cmd = "patch"
```

Comment:
Let's try just changing this to `patch` below and not use a variable (i.e. remove the `/usr/bin`). If it goes 💥 
I'll revert it.

Reply:
I got rid of the `cmd` variable. Did you mean also get rid of `with_system_path`?

Reply:
@sjackman Yep; make this just remove `/usr/bin/` and the variable usage.

Reply:
Done. I also replaced `IO.popen` with `Utils.popen_write`.